var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -29,6 +29,7 @@ class Boost(Package):
     license("BSL-1.0")
 
     version("develop", branch="develop", submodules=True)
+    version("1.88.0", sha256="46d9d2c06637b219270877c9e16155cbd015b6dc84349af064c088e9b5b12f7b")
```

Comment:
SHA confirmed.

Reply:
Just in case you haven't seen it, these are checked by CI nowadays: https://github.com/spack/spack/actions/runs/14591316532/job/40927032888?pr=50158.

Reply:
I did not know that. I've admittedly not done a very good job keeping up with the changes in the automated CI. It's good that they are doing that check.